docs/storing-querying/querying-data/index.md

```diff
@@ -0,0 +1,300 @@
+---
+title: "Querying Snowplow data"
+sidebar_label: "Querying data"
+sidebar_position: 3
+---
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+## Basic queries
+
+You will typically find most of your Snowplow data in the `events` table. If you are using Redshift or Postgres, there will be extra tables for [self-describing events](/docs/understanding-your-pipeline/events/index.md#self-describing-events) and [entities](/docs/understanding-your-pipeline/entities/index.md) — see [below](#self-describing-events).
+
+:::note
+
+Database and/or schema name will depend on your configuration, but we will use `atomic` as the schema name in the examples below.
+
+:::
+
+Please refer to [the structure of Snowplow data](/docs/understanding-your-pipeline/canonical-event/index.md) for the principles behind our approach, as well as the descriptions of the various standard columns.
+
+:::tip Data models
+
+Querying the `events` table directly can be useful for exploring your events or building custom analytics. However, for many common use cases it’s much easier to use our [data models](/docs/modeling-your-data/modeling-your-data-with-dbt/index.md), which provide a pre-aggregated view of your data.
+
+:::
+
+The simplest query could look like this:
+
+```sql
+SELECT * FROM atomic.events
+WHERE event_name = 'page_view'
+```
+
+:::caution
+
+With large data volumes (read: any production system), you should always include a filter on the partition key (normally, `collector_tstamp`), for example:
+
+```sql
+WHERE ... AND collector_tstamp > '2023-10-23'
+```
+
+This ensures that you read from the minimum number of partitions necessary, making the query run much faster.
```

Comment:
I can't do a suggestion because of the backticks but:

```sql
WHERE ... AND collector_tstamp between timestamp '2023-10-23' and timestamp '2023-11-23'
```

This ensures that you read from the minimum number of (micro-)partitions necessary, making the query run much faster and reducing compute cost (where applicable).